datafusion/src/execution/context.rs

```diff
@@ -982,6 +1008,14 @@ pub struct SessionState {
     pub runtime_env: Arc<RuntimeEnv>,
 }
 
+/// Default session builder using the provided configuration
+pub fn default_session_builder(config: SessionConfig) -> SessionState {
```

Comment:
Since there is already a `SessionContext::with_config_rt` maybe we could name this function `SessionState::with_config` and the existing `SessionState::with_config` to `SessionState::with_config_rt` for consistency

Reply:
Done